site/app/templates/submission/EditTeamName.twig

```diff
@@ -4,7 +4,10 @@
 {% block body %}
     <p class="std-margin">
         Please fill in an appropriate name for your team if you would like.
+        <br>
+        Team names are limited to 30 characters
     </p>
+    
     <div class="flex-col" id="team-name-modal">
         <input id="team-name-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
         <div class="edit_user_row">
```

Comment:
You can put the attribute `maxlength` https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength to perform a client side check that the input length is correct. You could also pass the MAX_LENGTH as a variable so that the front end backend will always be synced to the same constraint. 

This twig file is rendered from this view `site/app/views/submission/TeamView.php` and gets imported via `site/app/templates/submission/Team.twig` 

The diff isn't letting me click it but its line 12, the input where the name is "team_name"